### PR TITLE
chore: remove example crate

### PR DESCRIPTION
**Pull request by sripwoud**

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: docs
 on:
   push:
     branches: [main]
-    paths: ['src/**/*.rs', 'Cargo.toml', '.github/workflows/docs.yml']
+    paths: ['crates/**/*.rs', '**/Cargo.toml', '.github/workflows/docs.yml']
```
Comment (sripwoud) [inline .github/workflows/docs.yml:5]: I noticed #5 didn't trigger the docs workflow while it should have

Comment (cedoor) [reply .github/workflows/docs.yml:5]: Right!

**Comment by cedoor** [comment]
@sripwoud we need to update the Readme file as well (the list of crates).

**Comment by sripwoud** [comment]
> @sripwoud we need to update the Readme file as well (the list of crates).

#7

**Pull request merged**